<launch>
<!--<node pkg="tf" type="static_transform_publisher" name="imu_transform" args="0 0 0 -1.570796 0 3.14159265 base_link imu_link 100"/>-->
<node pkg="tf" type="static_transform_publisher" name="imu_transform" args="0 0 0 0 0 0 base_link imu_link 100"/>
<node name="serial_node"        pkg="rosserial_python"      type="serial_node.py">
	<param name="port"              type="string"               value="/dev/ttyACM0"/>
	<param name="baud"              type="int"                  value="57600"/>
	</node>
	
	<!--<include file="$(find kcctslam)/launch/hokuyo_urg.launch"/>-->
	<include file="$(find kcctslam)/launch/velodyne_2d.launch"/>
	<node pkg="kcctslam" type="robot_model_node" name="robot_model_node" output="screen"/>
 <node name="teensy_handler" pkg="kcctslam" type="teensy_handler"/>
</launch>
